itest/litd_stateless_init_test.go

```diff
@@ -0,0 +1,62 @@
+package itest
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// testStatelessInitMode runs various authentication tests against a node that
+// is running in stateless-init mode.
+func testStatelessInitMode(_ context.Context, net *NetworkHarness,
+	t *harnessTest) {
+
+	// Set up a new node (charlie) in stateless init mode (create and
+	// unlock).
+	walletPassword := []byte("stateless")
+	charlie, adminMac := net.NewNodeWithSeed(
+		t.t, "Charlie", nil, walletPassword, false, true,
+	)
```

Comment:
Should we maybe add test coverage for what the intended behaviour is when attempting to use a remote lnd node + stateless init mode, as we've had users who have attempted that? Else feel free to disable setting both of them to true in the `NewNodeWithSeed` function.

Reply:
what would such a test look like though? cause lit would just fail to connect to lnd cause there would be no macaroon to authenticate itself

Reply:
> Else feel free to disable setting both of them to true in the NewNodeWithSeed function.

dont think we need to explicitly guard against the set-up though - the test will fail either way :)

Reply:
gonna merge this in the mean time & we can continue discussion offline with possible follow up 👍